cmd/medius/images/push.go

```diff
@@ -247,6 +238,74 @@ func (b *buildAndPublish) readArtifact(artifactReader http.ReadCloserWithChecksu
 	return file.Name(), nil
 }
 
+func (b *buildAndPublish) buildImages(entry *common.Entry) ([]v1.Image, []string, error) {
+	var images []v1.Image
+	var artifacts []string
+
+	rebuildNeeded, err := b.rebuildNeeded(entry)
+	if err != nil {
+		return nil, nil, err
+	}
+	if !rebuildNeeded && !b.Options.PublishImagesOptions.ForceBuild {
+		b.Log.Info("Nothing to do.")
+		return nil, nil, nil
+	}
```

Comment:
Didn't you want to move this to `Do`?

Reply:
Oh I thought you meant to:
```
	if errors.Is(b.Ctx.Err(), context.Canceled) {
		return nil, nil, b.Ctx.Err()
	}
```
Let me check it again.

Reply:
Fixed.